lab1.js

```diff
@@ -99,12 +96,22 @@ assert(sentence2 === 'chirp chirp chirp chirp chirp chirp chirp chirp chirp.',
 */
 
 var favoriteAnimals = ['elephant', 'penguin', 'eagle', 'camel'];
-var nextAnimal;
 
-// TODO: 12 points
-// Assign one of your favorite animals to nextAnimal using Math.random() to pick
+var nextAnimal = Math.random();
 
-// your code goes here
+if (nextAnimal < 0.25) {
+  nextAnimal = favoriteAnimals[0];
```

Comment:
Here, whole numbers are advantageous because you could pass that number in as the index of the array. For example:
myVar = the code that gives you a random number between 0 and 3
nextAnimal = favoriteAnimals[myVar]



> On Jul 20, 2015, at 7:11 PM, Clare Monahan <notifications@github.com> wrote:
> 
> In lab1.js:
> 
> >  
> > -// your code goes here
> > +if (nextAnimal < 0.25) {
> > +  nextAnimal = favoriteAnimals[0];
> Is the preferred method in industry to make things whole numbers? The mathematician in me love decimals.
> 
> —
> Reply to this email directly or view it on GitHub.
>